src/rvi.c

```diff
@@ -952,9 +952,9 @@ int rviValidateCredential( TRviHandle handle, const char *cred, X509 *cert )
  * Initialize the RVI library. Call before using any other functions.
  */
 
-TRviHandle rviInit ( char *configFilename )
+TRviHandle rviJsonInit ( char *configContent )
```

Comment:
Yes, I didn't add const, because for some reasons there wasn't const in rviInit call